digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -79,7 +80,15 @@ class RoadAddressService(roadLinkService: RoadLinkService) {
     val viiteRoadLinks = roadLinks.flatMap { rl =>
       val ra = addresses.getOrElse(rl.linkId, Seq())
       buildRoadAddressLink(rl, ra)
+  }
+    val newAnomalousRL = viiteRoadLinks.filter(rl => (rl.administrativeClass == State || rl.roadNumber > 0) && rl.id == 0)
+
+    val missedRL  = withDynTransaction {
+    RoadAddressDAO.getMissingRoadAddresses()
```

Comment:
1. We should rework this operation, maybe introduce a filler here (because later we will be using that way of working). A filler stub is already in the master branch.

2. I don't think it's beneficial - we anyway do the service testing on integration level. So let's just use a rollback operations around them.